src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -81,10 +81,16 @@ class DataExplorerVisualizations extends React.Component {
     return clickFunc;
   }
 
+  isFileButton = buttonConfig => buttonConfig.type === 'manifest' ||
+      buttonConfig.type === 'export' ||
+      buttonConfig.type === 'export-to-workspace';
+
   refreshManifestEntryCount = () => {
     if (this.props.dataExplorerConfig
       && this.props.dataExplorerConfig.buttons
-      && this.props.dataExplorerConfig.buttons.some(btnCfg => btnCfg.type === 'manifest' && btnCfg.enabled)) {
```

Comment:
I haven't checked with other commons' gitops but here `manifest` type is removed, will it affect other commons if it's `manifest`?

Reply:
I made a function to determine if a button has anything to do with files! `isFileButton`

Reply:
Oh I didn't see the that. 😆   looks good to me!